src/nvim/buffer_defs.h

```diff
@@ -79,6 +79,11 @@ typedef struct {
 // Mask to check for flags that prevent normal writing
 #define BF_WRITE_MASK   (BF_NOTEDITED + BF_NEW + BF_READERR)
 
+
+// values for w_popup_flags.
+#define POPF_HIDDEN     0x02    // popup is not displayed
```

Comment:
"HIDDEN" is not something we use, so should be safe to remove?

Reply:
before removed. but I see there has an issue for add popup_hide popup_show relate function ..so I think can be add and maybe we will use it in next pr.